app/src/main/java/org/oppia/app/topic/revisioncard/RevisionCardViewModel.kt

```diff
@@ -49,18 +50,26 @@ class RevisionCardViewModel @Inject constructor(
   }
 
   private val revisionCardResultLiveData: LiveData<AsyncResult<RevisionCard>> by lazy {
-    topicController.getRevisionCard(topicId,subtopicId)
+    topicController.getRevisionCard(topicId, subtopicId)
   }
 
   private fun processExplanationLiveData(): LiveData<CharSequence> {
     return Transformations.map(revisionCardResultLiveData, ::processExplanationResult)
   }
 
-  private fun processExplanationResult(revisionCardResult: AsyncResult<RevisionCard>): CharSequence {
+  private fun processExplanationResult(
+    revisionCardResult: AsyncResult<RevisionCard>
+  ): CharSequence {
     if (revisionCardResult.isFailure()) {
-      logger.e("RevisionCardFragment", "Failed to retrieve Revision Card", revisionCardResult.getErrorOrNull()!!)
+      logger.e(
```

Comment:
Can we fit this in single line. I know ktlint does this formatting, but I want to confirm this with @rt4914 .

Reply:
Actually currently we cannot make this in single line because ktlint will fail on circle CI otherwise.

I think this will be with all the logger statements in the codebase so better keep it like this so that atleast everything passes on ktlint.

Reply:
Okay thanks for confirming @rt4914. In that case its good to merge.